src/utilities/units/units.ts

```diff
@@ -173,5 +173,8 @@ export function getValueFromArcsecString(formattedString: string): number | null
 }
 
 export function pixelToFluxDensityUnit(pixelUnit: string): string {
+    if (pixelUnit === "K") {
+        return "Jy";
```

Comment:
please check if the returned value is NaN or not. K to Jy conversion requires beam info and frequency info which may not be available from image header.

Reply:
The unit is hidden when the value is NaN.